features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_discount_for_whole_order.feature

```diff
@@ -0,0 +1,41 @@
+@applying_taxes
+Feature: Applying correct taxes for item units with a discount applied for all items in it
+    In order to pay proper amount when buying goods
+    As a Visitor
+    I want to have correct taxes applied to my order with a discount
```

Comment:
As above 👆🏼

Reply:
I'm resolving these comments, as they litter the PRek a bit, the main comment is enough